src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,21 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
+        {
+            var pipeLineExecutor = Builder.Build<PipelineExecutor>();
+            var incomingBehaviorTypes = pipeLineExecutor.Incoming.Select(registerStep => registerStep.BehaviorType).ToArray();
+            var outgoingBehaviorTypes = pipeLineExecutor.Outgoing.Select(registerStep => registerStep.BehaviorType).ToArray();
+            foreach (var outgoingBehaviorType in outgoingBehaviorTypes)
+            {
+                configurer.ConfigureComponent(outgoingBehaviorType, DependencyLifecycle.InstancePerCall);
```

Comment:
what if someone uses a custom configuration for a behavior within a feature? Isn't that overwritten here?